scripts/submit_emod3d.py

```diff
@@ -49,6 +49,8 @@ def main(
         nt = int(float(params.sim_duration) / float(params.dt))
 
         target_qconfig = get_machine_config(args.machine)
+        
+        retries = args.retries if hasattr(args, "retries") else None
```

Comment:
Why don't you just refactor out the args argument into this function? Seems simpler than the alternatives?

Reply:
This is not a valid English sentence. Please refactor.

Reply:
[You have recieved a review request from @jasonmotha ]